### ISSUE TITLE
Out-of-memory

### ISSUE THREAD

**Issue by ptahchiev** [bitnami/bitnami-docker-jenkins#47]
<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and we'll close it. We
will, however, reopen it if you later provide the information.
-------------------------------
    BUG REPORT INFORMATION
-------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

Hello guys,

I deployed your image on cloud.google.com (n1-standard-2 (2 vCPUs, 7.5 GB memory)). It works fine for a few days, but then every time I get out-of-memory like this one:
````
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 12288 bytes for committing reserved memory.
# Possible reasons:
#   The system is out of physical RAM or swap space
#   In 32 bit mode, the process size limit was hit
# Possible solutions:
#   Reduce memory load on the system
#   Increase physical memory or swap space
#   Check if swap backing store is full
#   Use 64 bit Java on a 64 bit OS
#   Decrease Java heap size (-Xmx/-Xms)
#   Decrease number of Java threads
#   Decrease Java thread stack sizes (-Xss)
#   Set larger code cache with -XX:ReservedCodeCacheSize=
# This output file may be truncated or incomplete.
#
#  Out of Memory Error (os_linux.cpp:2627), pid=12729, tid=0x00007fc5b3e6b700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_121-b13) (build 1.8.0_121-b13)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.121-b13 mixed mode linux-amd64 compressed oops)
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#

---------------  T H R E A D  ---------------

Current thread (0x00007fc5d01be000):  JavaThread "Monitoring thread for Clock Difference started on Sun May 07 12:57:45 UTC 2017" daemon [_thread_new, id=29402, stack(0x00007fc5b3d6b000,0x00007fc5b3e6c000)]

Stack: [0x00007fc5b3d6b000,0x00007fc5b3e6c000],  sp=0x00007fc5b3e6a860,  free space=1022k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
V  [libjvm.so+0xac703a]  VMError::report_and_die()+0x2ba
V  [libjvm.so+0x4fc7eb]  report_vm_out_of_memory(char const*, int, unsigned long, VMErrorType, char const*)+0x8b
V  [libjvm.so+0x923c43]  os::Linux::commit_memory_impl(char*, unsigned long, bool)+0x103
V  [libjvm.so+0x923d0c]  os::pd_commit_memory(char*, unsigned long, bool)+0xc
V  [libjvm.so+0x91d7ca]  os::commit_memory(char*, unsigned long, bool)+0x2a
V  [libjvm.so+0x9220ff]
~                                                                                                                                                                                                                                            
~                                        
````

Can you tell me how to edit the memory settings?

**Comment by javsalgar** [comment]
Hi, 

Please note that this Github page is for our Bitnami Jenkins container. Could you please post your issue in community.bitnami.com so we can help you there?
